### ISSUE TITLE
Error In ProbeJS Generated Script (jsconfig.json) inside server scripts folder

### ISSUE THREAD

**Issue by terrarian10** [Prunoideae/ProbeJS#71]
Error is the following

```
Specified 'include' paths were '["./src/**/*",null]' and 'exclude' paths were '[]'.ts
JSON schema for a JavaScript project using TypeScript tooling

```
However, the problem does not occur for the startup scripts folder jsconfig

**Comment by terrarian10** [comment]
ProbeJS Version: 7.0.0
Minecraft Version: 1.20.1
Forge Version: 47.3.1

**Comment by MeiNanita** [comment]
Got a similar issue with the latest probeJS-version.
Basically the src-folder inside the startup-folder is completely empty which I guess might be the issue.

So far I'm switching back to version 6.0.1 for now since that one works.

**Comment by Prunoideae** [comment]
You will need to place scripts in the `src/` folder.

**Comment by Prunoideae** [comment]
I'm currently rewriting ProbeJS to make it more like a true JavaScript project. It's targeting 1.21. The codegen is done and released on 1.20.1 for testing and feedback. Due to the codebase changes, stuff like QoL and other fixes (so you won't get confused if you don't know JS or TS) will be in 1.21.

**Comment by terrarian10** [comment]
Thank You! That Worked
